src/locale/locales/ca/messages.po

```diff
@@ -922,7 +922,7 @@ msgstr "Configura els filtres de continguts per la categoria: {0}"
 
 #: src/components/moderation/ModerationLabelPref.tsx:116
 msgid "Configured in <0>moderation settings</0>."
-msgstr ""
+msgstr "Configurat a <0>configuració de moderació</0>"
```

Comment:
I don't know if it matters here, but the full stop/period at the end is missing.